npm/generate_katas_content.js

```diff
@@ -566,6 +566,42 @@ function generateKataContent(path, globalCodeSources) {
   return kata;
 }
 
+function validateIdsUniqueness(katas) {
+  console.log("Validating IDs uniqueness across all katas");
+  const allIds = new Set();
+  const assertUniqueness = (id) => {
+    const isIdUnique = !allIds.has(id);
+    if (!isIdUnique) {
```

Comment:
This double negation makes my head spin :) May be name it alreadySeen or similar... Up to you.

Reply:
Renamed the variable to `idAlreadyExists` :)